src/icon/styles.scss

```diff
@@ -42,3 +42,58 @@
   inset-block-start: 0px;
   inset-inline-end: -3px;
 }
+
+/*
+Mirror all icons that are assymetrical or indicate a passage of 
+time when the icon direction is RTL. For browsers that do not 
+support the :dir() pseudoclass provide a fallback that depends 
+on the global document dir attribute.
+*/
+/* stylelint-disable plugin/no-unsupported-browser-features */
+@supports selector(:dir(rtl)) {
```

Comment:
I think it is already time to introduce a reusable mixin, similar to this one: https://github.com/cloudscape-design/components/blob/b958f52a3f68fa733386315e9d283f90c8d8e54e/src/internal/styles/motion/mixins.scss#L10-L22

Reply:
Agreed!